airflow/config_templates/default_airflow.cfg

```diff
@@ -271,9 +271,6 @@ access_logfile = -
 error_logfile = -
 
 # Expose the configuration file in the web server
-# This is only applicable for the flask-admin based web UI (non FAB-based).
-# In the FAB-based web UI with RBAC feature,
-# access to configuration is controlled by role permissions.
```

Comment:
If this is only applicable to the old UI why do we keep the setting?

Reply:
In 1.10.2, I remember I found that config was hidden even if for Admin. So I suspect that `expose_config` is also applicable for RBAC UI now (possibly due to some new commit). If you check the codebase, "expose_config" is still used somewhere.

So I didn't remove this.

Reply:
Double-checked: This `expose_config` item is used by the current UI (RBAC) now. We should keep it and remove the out-dated comment as I did.